source/resources/examples/typescript/restriction/AccountAddressRestrictionRemoveRestriction.js

```diff
@@ -38,7 +37,7 @@ const productAccount = nem2_sdk_1.Account.createFromPrivateKey(productPrivateKey
 const signedTransaction = productAccount.sign(transaction, networkGenerationHash);
 // replace with node endpoint
 const nodeUrl = 'http://api-xym-harvest-20.us-west-1.nemtech.network:3000';
-const repositoryFactory = new RepositoryFactoryHttp_1.RepositoryFactoryHttp(nodeUrl, networkType, networkGenerationHash);
+const repositoryFactory = new nem2_sdk_1.RepositoryFactoryHttp(nodeUrl, networkType, networkGenerationHash);
```

Comment:
probably don't need the `networkType` and `networkGenerationHash` has it gets generated in the constructor